### ISSUE TITLE
Relative statistics

### ISSUE THREAD

**Issue by mugling** [CleverRaven/Cataclysm-DDA#15783]
#15765 starts changing the ammo definitions to be relative to the base type and  #15781 proposes a system for item variants. Both should cut down on duplication and allow for quick rebalancing of the various types.

An incidental UI improvement is possible in that it's now possible to display how ammo compares to the default - for example `9mm +P+` should have improved stats relative to `9mm`.

![example](https://cloud.githubusercontent.com/assets/12238606/13756756/5450bcc2-ea19-11e5-80f8-d3e01f89db5c.png)

An alternative form would be `Damage: 26 (+4) = 30`.

What should be the preferred form?

**Comment by ZombiesAreUs** [comment]
From the screenshot, at first glance I'd have trouble knowing if the damage was 30 or 34, so the second one would probably be better [or even 30 (26+4)].

**Comment by Rivet-the-Zombie** [comment]
>From the screenshot, at first glance I'd have trouble knowing if the damage was 30 or 34, so the second one would probably be better [or even 30 (26+4)].

I'm going to second this.

**Comment by mugling** [comment]
I'd like to try and unify this with the gun stats if possible.]

So for `9mm +P+` from a `+1 gun`
```Damage: 26 (+1) (+4) = 31```

**Comment by codemime** [comment]
I like it a lot. IMHO the most meaningful value (the sum) should go first. It could also make sense to use justification, increase the gaps (since we have so much free space on the right) and drop the brackets to get something like this:
```
Damage:    31   =   26 + 4 + 1
Pierce:     4
Recoil:   320   =   260 + 60
```

**Comment by illi-kun** [comment]
I agree with @ZombiesAreUs and @Rivet-the-Zombie that alternative form is clearer. Also, it could be useful to explain this equation somewhere in game help.
